client/src/components/User/styles.css

```diff
@@ -1,39 +1,80 @@
-
-.user_container {
-    margin-bottom: 75px;
+.user-container {
```

Comment:
There is a "problem" in CSS with parent of floated blocks.
If you use **float** for some element in your layout, you should use **clear** to cancel floating for the next element.
http://htmlbook.ru/css/clear
You can check this problem in DevTools. Hover on .user-container. Its height would be less the then it looks on the page.
To fix this you can use one of possible solutions:

- The simple and faster: add `overflow: hidden;` to **.user-container**
- Add this styles (it is often called like clearfix :smile:):
```
.user-container:after {
    content: '';
    clear: both;
    display: table;
}
```

Reply:
Thank you! Finally I decided to do not use float, but it is good to know for the future! ❤️